app/views/companies/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<% provide(:title, "All companies") %>
+<div class="row">
+  <div class="col-md-12 text-center">
+  	<br/><br/><br/>
+    <h4>Subscribed Companies Details</h4>
+    <hr class="star-primary"><br/>
+	<div class="row">
+	  <div class="panel panel-default">
+	    <div class="table-responsive">
+	      <table class="table table-bordered table-hover table-heading">
+	        <thead>
+	          <tr class="success">
+	            <th class="text-center">Company Name</th>
+	            <th class="text-center">Description</th>
+	            <th class="text-center">Address</th>
+	            <th class="text-center">Actions</th>
+	          </tr>
+	        </thead>
+	        <tbody class="companies-tab-pane">
+	          <% @companies.each do |company| %>
+	            <tr>
```

Comment:
@godfreym  for the `tbody` you can put the `text-center` class in the `tr` element and it will affect the `td` elements